### PR TITLE
Move to implicit usings

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: src/Polly.Specs/GlobalUsings.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿// Global using directives
```
Comment (martincostello) [inline src/Polly.Specs/GlobalUsings.cs:1]: IMHO using the `<Using Include="..." />` item syntax in the project file is nicer that having to create an extra `.cs` file.

You can also put them in the `Directory.Build.props` file and cascade them to all projects and use conditions to activate them (`IsTestProject == true`) etc.

File: src/Polly/Polly.csproj [hunk 2]
```diff
@@ -33,6 +33,7 @@
     <PackageReference Include="System.ValueTuple" Version="4.5.0" />
   </ItemGroup>
   <ItemGroup>
+    <Using Remove="System.Net.Http" />
```
Comment (martincostello) [inline src/Polly/Polly.csproj:36]: Did you mean to add this one?

Comment (SimonCropp) [reply src/Polly/Polly.csproj:36]: yeah thats a known issue with implicit usings. one of the default includes when u do `ImplicitUsings>enable` but it doesnt exist in older frameworks. u can see how common it is https://sourcegraph.com/search?q=context:global+%3CUsing+Remove%3D%22System.Net.Http%22+/%3E&patternType=standard&sm=1

File: src/Polly.Specs/GlobalUsings.cs (hunk 1 above)
Comment (SimonCropp) [reply src/Polly.Specs/GlobalUsings.cs:1]: `<Using Include="..." />`. ok cool. i have gone that approach in some projects. I will move this PR over

Comment (SimonCropp) [reply src/Polly.Specs/GlobalUsings.cs:1]:
```
You can also put them in the Directory.Build.props file and cascade them to all projects and use conditions to activate them (IsTestProject == true) etc.
```

will do

**Comment by SimonCropp** [comment]
@martincostello feedback addressed. note it will need a squash if u decide to merge it

**Review by martincostello — APPROVED** [review]

**Pull request merged**